commons/src/test/java/org/eclipse/kapua/commons/util/SystemUtilsTest.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *     Red Hat Inc
+ *******************************************************************************/
+package org.eclipse.kapua.commons.util;
+
+import org.eclipse.kapua.qa.markers.junit.JUnitTests;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import java.lang.reflect.Constructor;
+import java.net.URI;
+
+@Category(JUnitTests.class)
+public class SystemUtilsTest extends Assert {
+
+    @Test
+    public void privateConstrutorTest() throws Exception {
+        Constructor<SystemUtils> constructor = SystemUtils.class.getDeclaredConstructor();
+        assertEquals(constructor.isAccessible(), false);
+        constructor.setAccessible(true);
+        constructor.newInstance((Object[]) null);
+    }
+
+    @Test
+    public void getNodeUriTest() throws Exception {
+        try {
+            URI uri = new URI("tcp://localhost:1884");
+            assertEquals(SystemUtils.getNodeURI(), uri);
+        } catch (Exception ex) {
+            fail("The uri is incorrect!");
+        }
+    }
+
+    @Test
+    public void getNodeUriUdpTest() throws Exception {
+        try {
+            URI uri = new URI("udp://localhost:1884");
+            assertNotEquals(SystemUtils.getNodeURI(), uri);
+        } catch (Exception ex) {
+
+        }
+    }
+
+    @Test
+    public void getNodeUriNullTest() throws Exception {
+        try {
+            URI uri = new URI(null);
+            SystemUtils.getNodeURI();
+            fail();
+        } catch (Exception ex) {
+            //expected
+        }
+    }
+}
```

Comment:
Please add additional tests:
 - URI without tcp/udp part
 - URI without ":" part
 - URI without "//" part
 - URI without "localhost" part
 - URI with "local", "host", "loclahost", etc...part instead of "localhost"
 - URI with random string

Please create all of this tests in one scenario, create arrays with all the possibilities so we make tests as clean and concise as possible.